src/types/params/types.ts

```diff
@@ -21,6 +21,9 @@ export interface TypeFiltersParams {
   productionYear?: number
   imagesCountGroup?: string[]
 }
+export interface LeadTypeFiltersParams {
+  searchQuery?: string
```

Comment:
what is this required for? wouldn't one use `LeadQueryParams` instead? this type is not used anywhere, but only exported

Reply:
I added that like the `TypeFiltersParams` and use it in DH, but I just checked I don't really use it for the search but for the `filters `and `derivedFilters` in the params.ts, but we will need this later, but for now I will delete here and on DH too.